VehicleInterfaces/package.mo

```diff
@@ -13,11 +13,12 @@ package VehicleInterfaces "VehicleInterfaces Library (Version 1.2.4) - Interface
       showStatistics=true,
       allowDuplicate=true),
     preferredView="info",
-    version="1.2.4",
-    versionDate="2016-03-02",
+    version="1.2.5",
+    versionDate="2018-08-01",
     versionBuild=1,
     dateModified="2016-03-02",
```

Comment:
You might wanna update the dateModified as well. Looks funny otherwise.

Reply:
Both versionDate and dateModified are dummies at the moment. But as I could check so far we will really be able to release v1.2.5 in August 2018. So I will adapt both dates accordingly.